content/blog/go-sdk-road-to-2/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Modern Cloud Infrastructure in Golang  - The Road to 2.0"
+authors: ["evan-boyle"]
+tags: ["go", "golang", "aws", "gcp", "azure"]
+date: "2020-02-25"
+meta_desc: "Pulumi + Go is a powerful combo for your cloud-native infrastructure."
+meta_image: "pulumigo.png"
+---
+
+Here at Pulumi, everyone on our engineering team is a Gopher. We build our open-source [pulumi/pulumi engine](https://github.com/pulumi/pulumi) and our SaaS with Go. Pulumi has supported Go in preview for more than a year, but we've recently invested in making Go support for Pulumi great as part of [Pulumi 2.0]({{< relref "/blog/pulumi-2-0-roadmap#better-language-support" >}}).
+
+## What is Pulumi?
+
+Pulumi lets you use real languages to express your application’s infrastructure needs, using a powerful technique called “infrastructure as code.” Using infrastructure as code, you declare desired infrastructure, and an engine provisions it for you so that it’s automated, easy to replicate, and robust enough for demanding production requirements. Pulumi improves this process by leveraging real languages and making modern cloud infrastructure patterns, such as containers and serverless programs, first-class and easy.
+
+Automatically create, update, or delete cloud resources using Pulumi’s infrastructure as code engine, removing manual point-and-clicking in the cloud provider consoles, UIs, and ad-hoc scripts.
```

Comment:
One bit of feedback in general is that it'd be great to say here why Go specifically is a great language for infrastructure as code. This reads as a very generic overview to Pulumi and it'd be nice to emphasize Go a little bit more FWIW.

Reply:
Updated.